src/coreclr/src/jit/hwintrinsiccodegenarm64.cpp

```diff
@@ -556,14 +585,25 @@ void CodeGen::genHWIntrinsic(GenTreeHWIntrinsic* node)
 
             case NI_Vector64_Create:
             case NI_Vector128_Create:
-                if (intrin.op1->isContainedFltOrDblImmed())
+            case NI_AdvSimd_DuplicateToVector64:
+            case NI_AdvSimd_DuplicateToVector128:
+            case NI_AdvSimd_Arm64_DuplicateToVector64:
```

Comment:
>NI_AdvSimd_Arm64_DuplicateToVector64 [](start = 17, length = 36)

I assume you will have to delete this as well since you have remove `DuplicateToVector64` from AdvSimd.ARM64.

Reply:
Yes. Fixed locally.